pkgs/applications/audio/dirt/default.nix

```diff
@@ -12,10 +12,14 @@ stdenv.mkDerivation rec {
   configurePhase = ''
     export DESTDIR=$out
   '';
+  postInstall = ''
+    cp -r samples $out/samples
```

Comment:
Are you sure that's the intended location? I'd expect it somewhat deeper, e.g. somewhere under ``$out/share/``.